### PR TITLE
feat(go): Runtime method and static invoke support

### PR DESCRIPTION
**Pull request by MrArnoldPalmer**
Adds support to the go runtime for class method and static method
invocation. Adds codegen to jsii-pacmak to call the new runtime
functions where appropriate.

fix: #2127

---

By submitting this pull request, I confirm that my contribution is made under the terms of the [Apache 2.0 license].

[Apache 2.0 license]: https://www.apache.org/licenses/LICENSE-2.0

### CODE REVIEW

File: packages/@jsii/go-runtime/jsii-experimental/runtime.go [hunk 1]
```diff
@@ -52,8 +52,48 @@ func Create(fqn FQN, args []interface{}, interfaces []FQN, overrides []Override,
...
+			JsiiInstanceId: refid,
+		},
+	})
+
+	if err != nil {
+		panic(err)
+	}
+}
+
+func SInvoke(fqn FQN, method string, args []interface{}, returns interface{}) {
```
Comment (RomainMuller) [inline packages/@jsii/go-runtime/jsii-experimental/runtime.go:84]: Couls we call this `InvokeStatic` instead? I get `go` likes terse... but I like descriptive?

**Review by RomainMuller — APPROVED** [review]

**Comment by MrArnoldPalmer** [comment]
@Mergifyio update

**Comment by MrArnoldPalmer** [comment]
@Mergifyio update

**Comment by MrArnoldPalmer** [comment]
@Mergifyio update

**Pull request merged**